vivarium/simulator/states.py

```diff
@@ -0,0 +1,316 @@
+from typing import Optional, List, Union
+from enum import Enum
+
+import matplotlib.colors as mcolors
```

Comment:
Not related to this PR, but I just notice that this requires `matplotlib`. We should either:

- Add matplotlib to the `requirements.txt` (if not already there)
- Or: find a way to do the same thing without using matlplotlib. The `mcolors` function is I think just used to convert colors between an vector RGB format (used in State) and a string format (used in the config). Maybe there is a way to do this without relying on matlplotlib.

If possible, I think the second option is better, because imposing to install matlplotlib (which is quite a big lib) just for this. We can discuss it and do it later though, you can just create a new issue atm

Reply:
There is actually a deeper issue here, see my other comment below

Reply:
I agree with you only using matplotlib for this isn't really useful. But it is needed to launch the interface with panel anyway I am not wrong. That's why it was already in the requirements.txt.